patch-package
--- a/node_modules/antd/lib/version/compatible.js
+++ b/node_modules/antd/lib/version/compatible.js
@@ -7,5 +7,5 @@
 // eslint-disable-next-line import/no-mutable-exports
 let antd = {};
 // @ts-ignore
-(0, _warning.default)(parseInt(_react.default.version, 10) >= 16, 'antd', 'An old version of React is not compatible with antd 5.x');
+(0, _warning.default)(true, 'antd', 'An old version of React is not compatible with antd 5.x');
 var _default = exports.default = antd;
